src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -168,13 +169,35 @@ const EvmEventParamData: FC<{
           />
         )
       }
+
+      const commonEvmContractDecimals = 18
+      const maybeParsedBaseUnits = fromBaseUnits(param.value as string, commonEvmContractDecimals)
+      if (!maybeParsedBaseUnits.startsWith('0.0000')) {
+        // Don't parse suspiciously low values.
+        return (
+          <Tooltip
+            arrow
+            placement="top"
+            title={param.value as string}
```

Comment:
I would format a value in a tooltip

```
           title={t('common.valueLong', {
              ...getPreciseNumberFormat(param.value as string),
            })}
```

Reply:
discussed during the meeting - this is intentional

Reply:
I wanted to keep the raw number in case our decimals are wrong, but happy to add decimal separators.

before:
```
Display: 1,292.0000001 ROSY
Tooltip: 1292000000100000000000
```

your proposal
```
Display: 1,292.0000001 ROSY
Tooltip: 1,292,000,000,100,000,000,000
```

meeting proposal:
```
Display: 1,292.000.. ROSY
Tooltip: 1,292.0000001 ROSY   // I'd like to keep raw value
```

Reply:
Added your proposal